ray-operator/hack/local_deploy.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+PROJECT_ROOT="${SCRIPT_DIR}/../.." # hack/ -> ray-operator/ -> kuberay/
+
+# Path to your Helm chart, relative to the PROJECT_ROOT (kuberay/)
+HELM_CHART_RELATIVE_PATH="helm-chart/kuberay-operator"
```

Comment:
I think in a previous comment we said this variable is not needed.  Line 11 can just be
```
HELM_CHART_PATH="${PROJECT_ROOT}/helm-chart/kuberay-operator"
```